   LOG VENPRO 2017-09-27
_________________________________________________________
---------------------------------------------------------

2017-09-27 17:58:24 : VENPRO iniciado
2017-09-27 17:58:32 : pru.103. False
2017-09-27 17:58:32 : pru.104. False
2017-09-27 17:58:32 : pru.123. False
2017-09-27 17:58:34 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 17:58:35 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 17:58:35 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 17:59:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 17:59:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 17:59:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:0:28 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:0:28 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:0:29 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:1:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:1:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:1:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:2:24 : Error descarga. No se puede escribir datos de en la conexión de transporte: Se ha forzado la interrupción de una conexión existente por el host remoto.
2017-09-27 18:2:24 : Error download(). Se ha terminado la conexión: Error inesperado de recepción.
2017-09-27 18:2:24 : Error descargar_archivo(). No se pudo descargar el archivo[TRPER20141110A0.XML][TIENDA: 104-Las Begonias]
2017-09-27 18:2:25 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:3:22 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No se pueden leer los datos de la conexión de transporte: Se ha forzado la interrupción de una conexión existente por el host remoto. ---> System.IO.IOException: No se pueden leer los datos de la conexión de transporte: Se ha forzado la interrupción de una conexión existente por el host remoto. ---> System.Net.Sockets.SocketException: Se ha forzado la interrupción de una conexión existente por el host remoto
   en System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   en System.Net.PooledStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   en System.Net.CommandStream.ContinueCommandPipeline()
   en System.Net.CommandStream.SubmitRequest(WebRequest request, Boolean async, Boolean readInitalResponseOnConnect)
   en System.Net.FtpWebRequest.TimedSubmitRequestHelper(Boolean async)
   en System.Net.FtpWebRequest.SubmitRequest(Boolean async)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:3:24 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:3:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:3:24 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:4:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:4:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:4:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:4:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:5:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:5:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:5:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:5:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:6:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:6:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:6:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:6:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:7:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:7:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:7:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:7:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:8:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:8:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:8:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:8:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:9:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:9:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:9:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:9:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:10:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:10:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:10:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:10:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:11:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:11:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:11:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:11:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:12:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:12:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:12:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:12:26 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:13:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:13:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:13:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:13:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:14:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:14:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:14:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:14:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:15:23 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:15:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:15:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:15:24 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//123/pcmaster/20170927. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 540
2017-09-27 18:16:22 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:17:22 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:18:23 : Error import_archivo(). [E:\VENPRO\VENPRO\bin\Debug\import\\104\TRPER20141110A0.XML]. System.IO.IOException: El proceso no puede obtener acceso al archivo porque está siendo utilizado en otro proceso.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.__Error.WinIOError()
   en System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   en System.IO.File.Move(String sourceFileName, String destFileName)
   en VENPRO.Form1.import_archivo(String fecha_venpro, Int64 codtienda, String ruta) en E:\VENPRO\VENPRO\Form1.cs:línea 1666
2017-09-27 18:18:26 : VENPRO Cerrado.
